### DISCUSSION TITLE
isolate just one voice

### DISCUSSION THREAD

**Discussion by OnivalSouza** [Anjok07/ultimatevocalremovergui#1409] [Q&A]
In the song Sound of Silence, by Simon and Garfunkel, how can you isolate just one of the voices? Or Simon or Garfunkel?
which models to use?
thanks
*
see here.
*
https://youtu.be/k0Xrll79Sxk?si=UTCZfOsL8VgAEUVr

**Comment by OnivalSouza** [comment]
Estou triste! Fiz uma pergunta há tempos e até hoje ninguem soube me dar alguma resposta! (pt-br)
I am sad! I asked a question a while ago and to this day no one has been able to give me an answer!

**Reply by LRonX** [reply]
Im not sure this would 100% work, BUT-- when creating a 2nd vocal track (using the split vocals option with the karaoke models) you might be able to achieve that.
From my experience so far- some tracks were separated successfully, some were not
So give it a try, but remember to take this with a grain of salt

**Reply by OnivalSouza** [reply]
> Não tenho certeza se isso funcionaria 100%, MAS-- ao criar uma segunda faixa vocal (usando a opção de vocais divididos com os modelos de karaokê) você pode conseguir isso. Pela minha experiência até agora, algumas faixas foram separadas com sucesso, outras não. Então, tente, mas lembre-se de levar isso com um grão de sal

thank you for the answer. I'll try it later!

**Comment by FlashlightET** [comment]
what i've done with music where the different voices are panned, is i pan the audio so that the desired vocal is in the center then run it through multiple karaoke models (specifically mdx net karaoke 1&2 and vr architecture bve) then i go through the three tracks and remove any leaking vocal from the different outputs (by silencing or center isolating on the individual outputs) and make a max spec of those
